bundle/manifests/collector_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -61,6 +61,48 @@ spec:
       labels:
         service: collector
         severity: Warning
+    - alert: ElasticsearchDeprecation
+      annotations:
+        message: The OpenShift Elasticsearch Operator is deprecated and is planned
+          to be removed in a future release. Red Hat provides bug fixes and support
+          for this feature during the current release lifecycle, but this feature
+          no longer receives enhancements. As an alternative to using the OpenShift
+          Elasticsearch Operator to manage the default log storage, you can use the
+          Loki Operator.
+        summary: Detected Elasticsearch as the on cluster storage which is being deprecated
+          in a future release.
+      expr: |
+        sum(min_over_time(container_cpu_usage_seconds_total{container=~'elasticsearch'}[5m])) > 0
+      for: 10m
+      labels:
+        service: storage
+        severity: Warning
+    - alert: FluentdDeprecation
+      annotations:
+        message: Fluentd is deprecated and is planned to be removed in a future release.
+          Red Hat provides bug fixes and support for this feature during the current
+          release lifecycle, but this feature no longer receives enhancements. As
+          an alternative to Fluentd, you can use Vector instead.
+        summary: Detected Fluentd as the collector which is being deprecated in a
+          future release.
+      expr: |
+        sum(min_over_time(container_cpu_usage_seconds_total{image=~'.+fluentd.+'}[5m])) > 0
```

Comment:
Same here regarding NS.  My recollection is for mCLF we only allow fluentd as part of a legacy deployment so we can restrict it there.  We should also have consistent labels going back to 5.5 that would allow us to further narrow this to our pods